app/services/spotlight/carrierwave_file_resolver.rb

```diff
@@ -0,0 +1,8 @@
+module Spotlight
+  # Used by RIIIF to find files uploaded by carrierwave
+  class CarrierwaveFileResolver < Riiif::AbstractFileSystemResolver
+    def pattern(id)
+      Spotlight::Masthead.find(id).image.file.file
```

Comment:
Just mastheads for now? Can we use `Spotlight::FeaturedImage` and get it for thumbnails too?